sdk/core/Azure.Core.TestFramework/src/RecordedTestSanitizer.cs

```diff
@@ -53,6 +53,15 @@ public void AddJsonPathSanitizer(string jsonPath, Func<JToken, JToken> sanitizer
             JsonPathSanitizers.Add((jsonPath, sanitizer ?? (_ => JToken.FromObject(SanitizeValue))));
         }
 
+        public void ReplaceHostInUri(string replacementHostName)
```

Comment:
Sad panda.

Reply:
I thought this was what you asked for?

Reply:
Ah, the fact  that we are doing it at all.